istioctl/cmd/internal-debug.go

```diff
@@ -83,22 +146,17 @@ By default it will use the default serviceAccount from (istio-system) namespace
 				}
 			}
 			var xdsRequest xdsapi.DiscoveryRequest
-			var namespace, serviceAccount string
+			var namespace, serviceAccount, resourceName string
 			if len(args) > 1 {
-				podName, ns, err := handlers.InferPodInfoFromTypedResource(args[1],
-					handlers.HandleNamespace(namespace, defaultNamespace),
-					kubeClient.UtilFactory())
-				if err != nil {
-					return err
-				}
-				pod, err := kubeClient.CoreV1().Pods(ns).Get(context.TODO(), podName, metav1.GetOptions{})
-				if err != nil {
-					return err
+				podName, ns, svcAccount, perr := PreHandleForXdsRequest(args[1], namespace, kubeClient)
```

Comment:
As mentioned, we should take a single parameter. That parameter should be sent to istio directly. For example, `internal-debug foo-bar?k=v` will send a request to `foo-bar?k=v`. Without all of this processing

Reply:
okay, got it, not just config_distribution, but all these. Fixing it!

Reply:
@howardjohn in fact, I am very agree with your opinion, this make the `istioctl` tool has unified format and style when user use it. It will be a great experience for user. However, I think there are many update for other commands.